crates/comments/src/directive.rs

```diff
@@ -92,5 +121,11 @@ mod test {
             parse_comment_directive("#  fmt:  skip file"),
             format_skip_file
         );
+
+        assert_eq!(parse_comment_directive("# Generated by:"), format_skip_file);
+        assert_eq!(
+            parse_comment_directive("# foo\n# @generated\n # bar"),
```

Comment:
This example doesn't make sense to me because we would never get more than 1 comment as a slice of `text` coming through here